dpinger.c

```diff
@@ -840,7 +855,7 @@ get_length_arg(
 static void
 usage(void)
 {
-    fprintf(stderr, "Dpinger version 3.1\n\n");
+    fprintf(stderr, "Dpinger version 3.1.1\n\n");
```

Comment:
I don't think you need to bump version here since 3.1 was never released

Reply:
Got it